packages/neo-one-smart-contract-test/src/withContracts.ts

```diff
@@ -10,6 +10,39 @@ import { createNode } from './createNode';
 
 export { Contract, WithContractsOptions, TestOptions };
 
+// tslint:disable-next-line no-any
+export const getContractFromOptions = (options: any, smartContractName: string): SmartContractAny => {
+  const smartContract: SmartContractAny = options[smartContractName];
+
+  // tslint:disable-next-line: strict-type-predicates
+  if (smartContract === undefined) {
+    const re = new RegExp(smartContractName.toLowerCase(), 'g');
+    const matches = Object.keys(options).filter((key) => key.toLowerCase().match(re));
+    if (matches.length) {
+      // tslint:disable-next-line: no-console
```

Comment:
Is there a `monitor` around we could pass in and `monitor.logError(...)` instead of calling console.error directly? :thinking:

Reply:
I couldn't find any `monitor` available to the calling context:
`packages/neo-one-smart-contract-lib/src/__data__/tokenUtils.ts`